client/widgets/CArtifactHolder.cpp

```diff
@@ -206,12 +206,17 @@ void CHeroArtPlace::clickLeft(tribool down, bool previousState)
 bool CHeroArtPlace::askToAssemble(const CArtifactInstance *art, ArtifactPosition slot,
                               const CGHeroInstance *hero)
 {
-	assert(art != nullptr);
-	assert(hero != nullptr);
-	std::vector<const CArtifact *> assemblyPossibilities = art->assemblyPossibilities(hero);
+	assert(art);
+	assert(hero);
+	bool assembleEqipped = true;
+	if(slot >= GameConstants::BACKPACK_START)
+	{
+		assembleEqipped = false;
+	}
```

Comment:
Probably outside scope of this pull request so feel free to ignore, but:
- create method `ArtifactPosition::isBackPackSlot() { return num >= GameConstants::BACKPACK_START; }`
- use this method where applicable instead of this magic constant check

Reply:
good idea, i'll do it